content/about/notes/0.8.md

```diff
@@ -20,7 +20,8 @@ increases effective scalability and should eliminate spurious 404 errors. [TBD:
 - **Gateway for Ingress/Egress**. We no longer support combining Kubernetes Ingress specs with Istio route rules,
 as it has led to several bugs and reliability issues. Istio now supports a platform independent ingress/egress
 [Gateway](/docs/concepts/traffic-management/rules-configuration/#gateways) that works across
-Kubernetes and Cloud Foundry and works seamlessly with the routing rules.
+Kubernetes and Cloud Foundry and works seamlessly with the routing rules. The Istio Gateway supports SNI [Server Name Indication]
```

Comment:
Cloud Found4y both work seamlessly

Reply:
It's the ingress/egress that works.

Reply:
The sentence is not right:

"k8s and cloud foundry and works"

Reply:
Do we have a link to refer to users for use SNI in istio?

Reply:
Not yet, we can update later.

Reply:
There is nothing to do with SNI. 
I would call out SNI separately. Simply state that HTTPS access to external services will be auto configured based on SNI. 
Also state that Gateways support SNI based routing as well as serving certificate based on the SNI value.